casbin/model/policy.py

```diff
@@ -81,14 +83,39 @@ def has_policy(self, sec, ptype, rule):
 
         return rule in self[sec][ptype].policy
 
-    def add_policy(self, sec, ptype, rule):
+    def add_policy(self, sec, ptype, rule) -> bool:
```

Comment:
Maybe we don't need to think about type hint right now.

Reply:
Yeah but without type hint sometimes it's not easy to follow and regulate the program, especially for new developers.
Maybe we can add it later in a major release.

Reply:
@ffyuanda You can lead this to the whole library in another PR.

Reply:
@Zxilly Sure we can do that